common/src/main/java/org/apache/celeborn/common/client/MasterNotLeaderException.java

```diff
@@ -22,6 +22,7 @@
 import javax.annotation.Nullable;
 
 import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.tuple.Pair;
 
 public class MasterNotLeaderException extends IOException {
```

Comment:
the change in this class should be safe, but better to have a manual test to ensure that 0.4 client works well with main branch service

Reply:
Yes, it's compatible with 0.4~